requirements.tex

```diff
@@ -3,7 +3,12 @@ \section{Requirements} \label{sec:reqs}
 There are several different stakeholders of the \gls{LSST} \gls{DM} software products, each of whom may have different requirements for releases.
 In this section we give an overview of their respective requirements on  \gls{DM} software product releases.
 
-To be kept in mind that the content of a release, must be well understood and documented in the corresponding release note.
+It is important to identify the correct release policy during contraction phase, in order to have it consolidated when operations starts.
+
+Once in operations, some of the LSST subsystems mentioned here may not be relevant anymore, but the policy defined here shall still be applicable, since many of the stakeholders will still expect software releases to be managed properly. 
+
```

Comment:
In operations, some of the LSST subsystems may no longer exist, for example \gls{DM} will not exist per se in operations but \gls{DM} products, such as the Science Pipelines, will still exist under some other managerial structure. The policy defined here will still be applicable, since many of the stakeholders will still expect software releases to be managed in the same manner as during construction.

Reply:
This is paragraph is not just for operational (processing) software but is applicable in general. Infrastructural or other subsystems SW will still be used (and maintained) during operations even if DM (or T&S) are not LSST operational subsystems.